core/util.bzl

```
@@ -287,3 +288,28 @@ def expand_location(repository_ctx, string, labels, attr = None):
             fail("Internal error: Unknown location expansion command '{}'.".format(command), attr)
 
     return result
+
+def is_bazel_version_at_least(threshold):
+    """ Check if current bazel version is higer or equals to a threshold.
+
+    Args:
+        threshold: string: minimum desired version of Bazel
+
+    Returns:
+        treshold_met, from_source_version: bool, bool: tuple where
+        first item states if the treshold was met, the second indicates
+        if obtained bazel version is empty string (indicating from source build)
+    """
+    treshold_met = False
+    from_source_version = False
+
+    bazel_version = versions.get()
+    if not bazel_version:
+        from_source_version = True
+    else:
+        treshold_met = versions.is_at_least(threshold, bazel_version)
+
+    return (
+        treshold_met,
+        from_source_version,
+    )
```

Fix typo: `treshold` -> `threshold` 
```suggestion
    threshold_met = False
    from_source_version = False

    bazel_version = versions.get()
    if not bazel_version:
        from_source_version = True
    else:
        threshold_met = versions.is_at_least(threshold, bazel_version)

    return (
        threshold_met,
        from_source_version,
    )
```